reddit.js

```diff
@@ -2,4 +2,43 @@ var reddit = require("./data.json")
 
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+//forEach iterator for all titles
+
+reddit["data"].children.forEach(function (title) {
```

Comment:
Here, I'd advise that ```title``` is not the most suitable parameter name for this function. Since your ```forEach``` iterator is iterating over each *object* in the ```data.children``` array, perhaps 'post' or 'child' would be a more accurate representation of what you're dealing with here.